src/front/app/features/auth/auth.actions.js

```diff
@@ -74,12 +77,12 @@ export function login(loginService) {
                     response.toString().replace(/(<([^>]+)>)/ig, '')
                 );
                 if (responseJson.success && responseJson.token) {
-                    return Promise.resolve(responseJson.token);
+                    return Promise.resolve(responseJson);
                 }
                 return Promise.reject(response);
             })
-            .then((token) => {
-                dispatch(loginSuccess(token));
+            .then((result) => {
+                dispatch(loginSuccess(result));
             })
             .then(redirect)
```

Comment:
for redirect it`s better to use react-router-redux 
https://github.com/reactjs/react-router-redux#what-if-i-want-to-issue-navigation-events-via-redux-actions